changelog/fragments/ansible-helm-flags.yaml

```diff
@@ -0,0 +1,71 @@
+# entries is a list of entries to include in
+# release notes and/or the migration guide
+entries:
+  - description: >
+      (ansible/v1)(helm/v1) In order to align the Ansible/Helm based-projects with the Kubernetes nomenclatures the flags `--enable-leader-election` and `--metrics-addr` were deprecated and replaced by `--leader-elect` and `--metrics-bind-address` respectively.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "deprecation"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: (ansible/v1)(helm/v1) Replace the deprecated flags
+      body: >
+        Replace the flags `--enable-leader-election` and `--metrics-addr` which were deprecated with `--leader-elect` and `--metrics-bind-address` respectively.
+  - description: >
+      (ansible/v1)(helm/v1) Ensure that the `--health-probe-bind-address` port will be informed to the proxy.
+
+    # kind is one of:
+    # - addition
+    # - change
+    # - deprecation
+    # - removal
+    # - bugfix
+    kind: "bugfix"
+
+    # Is this a breaking change?
+    breaking: false
+
+    # NOTE: ONLY USE `pull_request_override` WHEN ADDING THIS
+    # FILE FOR A PREVIOUSLY MERGED PULL_REQUEST!
+    #
+    # The generator auto-detects the PR number from the commit
+    # message in which this file was originally added.
+    #
+    # What is the pull request number (without the "#")?
+    # pull_request_override: 0
+
+
+    # Migration can be defined to automatically add a section to
+    # the migration guide. This is required for breaking changes.
+    migration:
+      header: (ansible/v1)(helm/v1) Ensure that the `--health-probe-bind-address` port will be informed to the proxy.
+      body: >
+        Add the arg `--health-probe-bind-address=:8081` to the `config/default/manager_auth_proxy_patch.yaml`:
```

Comment:
This is different for ansible (port 6789). Make another changelog entry I guess?